src/org/sasylf/IDEProof.java

```diff
@@ -216,7 +216,7 @@ public void dispose() {
 	 * @return proof object or null
 	 */
 	public static IDEProof getProof(IAdaptable res) {
-		return proofs.get(IProjectStorage.Adapter.adapt(res));
```

Comment:
I don't think this file should be changed in this PR.

Reply:
That's right.  DOn't change it.

Reply:
Fixed